EIPS/eip-4844.md

```diff
@@ -177,10 +194,13 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
 The hash of a signed blob transaction should be computed as:
 
 ```python
-def signed_tx_hash(tx: SignedBlobTransaction) -> Bytes32:
-    return keccak256(BLOB_TX_TYPE + ssz.serialize(tx))
+def signed_tx_hash(tx: SignedBlobTransaction) -> Root:
+    domain = compute_transaction_domain(BLOB_TX_TYPE, node.cfg.chain_id)
+    return compute_signing_root(tx, domain)
```

Comment:
For the `signed_tx_hash`, as it already includes the signature (which already signs over `compute_transaction_domain`), I think it may be possible to just use plain `hash_tree_root`. TBD.